### DISCUSSION TITLE
Broken anchor links to headings with spaces in auto-generated Table of Contents for mediawiki pages

### DISCUSSION THREAD

**Discussion by asoltys** [community/community#41853] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

.mediawiki files with more than 3 headings have a Table of Contents that gets auto-generated and links to the various document headings.

Links to headings that have spaces in them are broken because the link in the ToC replaces spaces with underscores whereas the anchor itself has spaces replaced with hyphens.

Here is an example of a broken link that appears in the ToC: 

https://github.com/bitcoin/bips/blob/master/bip-0322.mediawiki#Detailed_Specification

And here's an example of a working link to the same heading: 

https://github.com/bitcoin/bips/blob/master/bip-0322.mediawiki#detailed-specification

**Comment by chlige** [comment]
We were recently informed by GitHub technical support that the automatically generated tabled of contents in MediaWiki pages is not supported due to security reasons.   See https://docs.github.com/en/communities/documenting-your-project-with-wikis/editing-wiki-content#supported-mediawiki-formats for details.

We were told by tech support that we should disable by adding `__NOTOC__` to the start of any MediaWiki page.
